### ISSUE TITLE
Portal 2 VR not working with Portal 2 Sixense Perceptual Pack

### ISSUE THREAD

**Issue by ALGOMOTH47** [Gistix/portal2vr#143]
i got the to run but when it gets to the main menu it View synonyms for immediately crashes

to get the game to run at all i had to modify some file possisens 
1. had to make a foldor named console
2. had to put portal2_product_1_widescreen.vtf, portal2_product_2_widescreen.vtf, startup_loading.vtf into this foldor files attached
[portal2_product_1_widescreen.zip](https://github.com/user-attachments/files/16479060/portal2_product_1_widescreen.zip)

note. to reppduce this i used the forked version shown here: https://github.com/stsichler/portal_reloaded_vr/releases/tag/v0.2.0p1-reloaded.2
I also used this pack commuity made patch of Portal 2 Sixense Perceptual Pack shown here: https://www.moddb.com/games/portal-2/downloads/portal-2-sixense-motionpack-post-february-2021-setup

**Comment by FoxbitDreamtail** [comment]
when I launch it by replacing my steam files for the mod I still had in steam and ran the linux native, it launched and then it kicked up ~5 l4d2 vr errors about invalid addresses while the game was running and closing them closed the game...

**Comment by ALGOMOTH47** [comment]
When I ran it I used the six senses vr patch with I assume is why it crashed on entry to a map as it is trying to run 2 open vr drivers (i think it's open vr although it may be openxr) or at least i think that's why

**Comment by FoxbitDreamtail** [comment]
it is OpenXR, yeah probably, I will look around github and see if someone put in the work to merge these 2 code bases.

**Comment by FoxbitDreamtail** [comment]
so... I found this https://github.com/CrossVR/Hydra-OpenVR
it's the codebase for using xr controllers through steamvr for those specific use cases where the proprietary hardware for this is emulated via steam vr input.

**Comment by ALGOMOTH47** [comment]
Yeah that's what I used.  If it could be adapted to work with portal2vr's code base for the one to one, scaling and portal rotation that would be really cool. As it is though it just crashes.  I have a huntch that it has to do with the player camera movement as that in p2vr is typed to the hmd but with sixsinse it's tied to the controller xyz and conflicts with the p2vr portal gun.

**Comment by FoxbitDreamtail** [comment]
How hard would it be to bind it to a toggle? Use the VR camera or the sixense camera? The ideal solution is rewriting everything from scratch using the preexisting code as reference, but I haven't the skill necessary to pull that off.

**Comment by Eddie-Baledy** [comment]
I genuinely hope to see something come from this honestly because I have always wanted to play this pack and was always so sad to never see the features implemented wihout requiring a device with sixense. qnq
